packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -99,7 +99,7 @@ function transformValueType(valueType: OTValueType): ValueType {
 }
 
 /**
- * Converts metric's timeseries to a list of TimeSeries, so that metric can be
+ * Converts metric's timeseries to a TimeSeries, so that metric can be
```

Comment:
Updated the comment here since a single TimeSeries is being returned. I think the discrepancy was made when this code was copied over.